src/lib.rs

```diff
@@ -227,4 +227,19 @@ mod tests {
         HWIClient::set_log_level(types::LogLevel::DEBUG).unwrap();
         test_enumerate();
     }
+
+    #[test]
+    #[serial]
+    fn test_toggle_passphrase() {
+        let devices = HWIClient::enumerate().unwrap();
+        for device in devices {
+            if device.device_type == "ledger" {
```

Comment:
ACK for the approach, but the ledger is not the only device with this problem. It's the only device **we test** with this problem.

https://hwi.readthedocs.io/en/latest/devices/index.html#support-matrix

Reply:
That is correct. I was hoping when we add new devices, we would modify the conditions at the same time.